README.md

```diff
@@ -207,18 +208,37 @@ Example of a partial CR:
 ```
 
 
-### Mounting binding information
+### Mounting and injecting binding information
+
+This specification allows for data to be mounted using volumes or injected using environment variables.  The best practice is to mount any sensitive information, such as passwords, since that will avoid accidently exposure via environment dumps and subprocesses. 
```

Comment:
Binding binary data (e.g. `.p12` cert for Kafka) as an environment variable might cause a pod failing to start and getting stuck in `CrashLoopBackOff`. In that case, binary data must be bound as a volume.

What do you think about highlighting that here? so operator providers know specify that they need to specify `bindAs: volume` or else they risk breaking the service binding for their operator and others in the same `ServiceBinding` CR.

Reply:
BTW, to see the pod error I'm referring to:
```
cat <<EOS |kubectl apply -f -
---
kind: Secret
apiVersion: v1
metadata:
  name: test-secret
data:
  # Key with a dummy PKCS #12 (.p12) cert
  DUMMY_KEY: >-
    MIIEVgIBAzCCBA8GCSqGSIb3DQEHAaCCBAAEggP8MIID+DCCA/QGCSqGSIb3DQEHBqCCA+UwggPhAgEAMIID2gYJKoZIhvcNAQcBMCkGCiqGSIb3DQEMAQYwGwQUq4Nyf5hZeejsHdLKF3ExTE8EMGwCAwDDUICCA6DzSgTq9TOAIe6pEirLJ+j3vvghZQbfQI+Vzy/tJqg3NQI7IQR89Pb7jfR6wuzKqjflDDqDdh7ay3lZtMfCq3vcjupQv6c8LQE2zBezgBtBZLVstPAA84pObLcjO3dALya5QCFvnoJIcZLR6dWH++QQtLHxg1rqVnYXxZjd3CQmirJFWY5+uO2nOYFmaYP6ISiiOc0vNMV4kbg8UdfBma9GY6TYT1y4QrjAYfM/fhivqjZ6JoWdIs6B3p/yIiXOItE+u/BicO0INTzg2Act4Pe8igFcAh9Yybir+3kFOBEE3GdnjddfCTxbrkLEzLnNxgr4BBndzhANxYQEaPncNHlpkhGEjqPP7mdJP72FiyyaioElbd2kFZac5aNvMBa53ZrXmI4iJdb8gODm6pfiNRn0rU8BqlpyRvAyXEMOInYmMs8sZjmYkUEaxj4PNMwfGS+1XbrboGGSk5Q/DhIQLhm3iHOcLHVfuZjWI4vm56n1YNUW/alFhhXKIxbKn1izujCnxGnVd0xQISehntu6YAiD5NXLq8g46FoSmpDNrhUYT/1c3O/3dtTVflwvYnPVMz2iebajdVtF1zRzJKQUhWUhdG0hRD+eQe1vQKe0JJhYu2Dke2pjjUaeEgrQL1iRqSwfH/HwvCCiEd+u7jOR28lJ0eENqPDINX4DDIZHrBw1W3w7U1sgw0GtdzhUUPz0/qWlO0RaD4mEaT1JY7p0FFMCDbg1dnEKGxx/xny8W2PF3M9KJO4z9qV23K6832kNBtVi4ixXaG7NP+dDmNhp681kBkRoO/0IE0mqBk1sps2+XT8z2lCyqLmsCHgZqca1CrJD1E19tYWM8LluqxDADzhDYSHDYOgz4LnQQLisq/QXoSWPGV52cBBFG9hfs0ItQn56daVdcRwfwvnE2ezOD2uiRjSjTfWWpEzL5FUCaIWmcs46B4/vZudryWmdVPSnxgfmtmNq60VO8SpzXZ+9/Oom91EsR/UwsKCqHUmFLxGQwVU5l9lyZjEPybaJIvpFrDPVFsn5Rxz5ysNbAxIX1U7MVe0MJnMTp0skfM3gPauehKXc2DW5U0McyoZB6tLpiK3HSKGkTXRrSkI/nSGrMyexIsuKBZYYzLhIHnkxNo/jYEjtZR73KfYX9zSbeDmqqULKoXGZHH8etO3MNZoMif1CIkJ6UWlO70MVP0hy5iWPYESwAEmJ+Zoqx2LGK/qWazimNG74cKV1lsIpXycd3eVSMD4wITAJBgUrDgMCGgUABBR+9aT95fPFF5H4PRQMOkhEddxxJwQU8Gk81TU6xjivACpOOyiMZ/sFJyoCAwGGoA==
type: Opaque
---
apiVersion: v1
kind: Pod
metadata:
  name: envfrom-secret
spec:
  containers:
  - name: envars-test-container
    image: nginx
    envFrom:
    - secretRef:
        name: test-secret
EOS
```

Reply:
that's a great point! will add it.

Reply:
added new commit.  thanks @navidsh